action.yaml

```diff
@@ -22,7 +22,6 @@ runs:
         NOCOLOR="\033[0m";
 
         apt-get -qq update
-
```

Comment:
Removing this line will be interpreted as putting combining these two lines https://github.com/JafarAbdi/moveit_task_constructor/runs/3690371184?check_suite_focus=true#step:6:17